package.json

```
@@ -2,7 +2,7 @@
   "name": "pulsar",
   "author": "Pulsar Community <noreply@pulsar-edit.com>",
   "productName": "Pulsar",
-  "version": "1.102.0-dev",
+  "version": "1.103.0",
```

```suggestion
  "version": "1.103.0-dev",
```

Follow-up stuff from release yesterday, this diff would help get the version bump back up to `master` branch and the Rolling releases.